guide/contributing/developer-guide.md

```diff
@@ -165,35 +245,143 @@ npm run test:e2e -- [opts]
+- `devDependencies` only contains what is required for testing or building into the final distributable
+
+#### Inspecting builds
+
+Every components' distributable code is ignored by git in every packages' respective **dist** directory. They must be built by Rollup
+before they can be inspected. Once built, view the code within either **index.es.js** (ECMAScript) or **index.cjs.js** (CommonJS) and
+assure none of the package's dependencies were unintentionally included. If they were, there may be an issue with how **package.json**
+or **rollup.config.js** is configured.
+
+<!-- TODO: add summary of how we name things -->
+
+### Manual tests
+
+To ensure all consumers of TDS will be able to consume and build their applications with a component, it must be compatible with a
+TELUS starter kit application. To test a yet-released component in your own application, we recommend the following techniques:
+
+#### Option 1: npm link
+
+`npm link` allows you to rapidly test a package via symlinks. To test with npm link, run these commands:
+
+```sh
+# TDS root directory
+npm run build -- @tds/your-component-name
+cd packages/ComponentName
+npm link
+```
+
+Once you have a package linked, it needs to be linked within your project. You will also need to manually add the component's
+package name to your project's **package.json**. Run these commands to add symlinks within your project's **node_modules** directory:
+
+```sh
+# Your project
+cd ~/path/to/your/project
+npm link @tds/component-name
+```
+
+Learn more about [npm link on npm](https://docs.npmjs.com/cli/link).
+
+#### Option 2: npm pack
+
+`npm pack` builds a tarball that can be installed in project via `npm install`; it also represents the final package that gets shipped to the npm registry. To set up a package tarball, run these commands:
+
+```sh
+# TDS root directory
+npm run build -- @tds/your-component-name
+cd packages/ComponentName
+npm pack
+```
+
+Once your tarball is built, install it directly within your project's **node_modules** directory:
+
+```sh
+# Your project
+npm install ~/path/to/tds/packages/ComponentName/tds-tarball.tgz
+```
+
+Learn more about [npm pack on npm](https://docs.npmjs.com/cli/pack).
+
+#### Testing in browsers
+
+Once your project is set up, be sure to test in all [supported browsers][faq-browsers] including:
+
+- Safari and Chrome on iOS
+- Chrome on Android
+- Internet Explorer 11
+
+Always test mobile devices first. While testing, perform the following checks at least:
+
+- Tests in [aXe][axe] or [Wave][wave] pass
+- Component renders fully in viewports from 320px width or greater
+- Component renders legibly when browser font size is adjusted from 16px to 32px
+- Interactive elements are keyboard navigable
+- Assistive technology can parse copy in a reasonable order
+- Animations are slowed or removed when [reduced motion][css-tricks-reduced-motion] is enabled
+- Focusable items are followed when using a screen magnifier
```

Comment:
As far as I know, this is out of the developer's control since programmatically focusing doesn't move the magnifier.

Reply:
I moved this to the Accessibility section above, and changed it to:

>- Desirable areas of interactive elements can be focused using a screen magnifier